next.config.mjs

```diff
@@ -1,4 +1,8 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {};
+const nextConfig = {
+  env: {
+    BASE_URL: process.env.BASE_URL,
```

Comment:
It's good to add .env.sample file on github and add all envs key there with blank values if something is insecure/privet e.g. BASE_URL can be filled out, it's not really insecure property.